### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by robertauer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+ARTIFACT_ID=k8s-loki
+MAKEFILES_VERSION=8.4.0
+VERSION=2.9.1-1
```
Comment (kahoona77) [inline Makefile:3]:
```suggestion
VERSION=5.23.0-0
```
start with a "-0", so that the first release has the "-1"

File: k8s/helm/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: k8s-loki
+description: A Helm chart for Kubernetes
```
Comment (kahoona77) [inline k8s/helm/Chart.yaml:3]: Maybe give it a little bit more specific description

File: Makefile (hunk 1 above)
Comment (robertauer) [reply Makefile:3]: But this should be the tool version, not the official Chart version, right? So `2.9.1-0` I think.

File: k8s/helm/Chart.yaml (hunk 2 above)
Comment (robertauer) [reply k8s/helm/Chart.yaml]: Done

**Review by kahoona77 — APPROVED** [review]

**Pull request merged**